assignment2/even_sum.rb

```diff
@@ -0,0 +1,42 @@
+# blocks
+
+a = [1, 2, 3, 4, 4, 5, 8, 16, 25];
+even_sum = 0
+
+#Blocks
+
+a.each {|i|
+    if (i % 2 == 0)
+        even_sum += i
+    end
+}
+
+puts even_sum
+
+#lambdas
+
+even_sum1 = -> (arr, even_sum = 0) {
```

Comment:
I think you can initialize `even_sum` variable inside the block instead of taking it as a 2nd param.